python/buku/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                buku
 version             4.9
-revision            0
+revision            1
```

Comment:
The installed files are not changing, so no need to bump the revision.

Reply:
Fixed!